src/System.Security.Permissions/ref/System.Security.Permissions.cs

```diff
@@ -0,0 +1,1108 @@
+namespace System.Security {
+  public abstract partial class CodeAccessPermission : System.Security.IPermission, System.Security.ISecurityEncodable, System.Security.IStackWalk {
+    protected CodeAccessPermission() { }
+    public void Assert() { }
+    public abstract System.Security.IPermission Copy();
+    public void Demand() { }
+    public void Deny() { }
+    public override bool Equals(object obj) { return default(bool); }
+//    public abstract void FromXml(System.Security.SecurityElement elem);
```

Comment:
What's the reason these are commented out?  We're waiting on some dependency to be satisfied?